### PR TITLE
section view

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
I'll rebase this branch to master once #42 is merged.
https://jira.sng.sk/browse/NSNG-67

Also I've created a new issue for redesign of item thumbnail.
https://jira.sng.sk/browse/NSNG-82

### CODE REVIEW

**Review by igor-kamil — CHANGES_REQUESTED** [review]
in section there should be also back button according the design
Snímka obrazovky 2022-11-08 o 15 46 03
the rest is good!

File: resources/js/bootstrap.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
+const { getActiveLanguage } = require('laravel-vue-i18n')
+
 window.axios = require('axios')
 
 window.axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest'
-window.axios.defaults.headers.common['X-locale'] = document.documentElement.lang
+window.axios.interceptors.request.use((config) => {
+    config.headers.common['X-locale'] = getActiveLanguage()
+    return config
+})
```
[the remark this answers is not in the document]
Comment (rastislav-chynoransky) [reply resources/js/bootstrap.js:9]: DRYness :)

**Comment by rastislav-chynoransky** [comment]
> in section there should be also back button according the design

I forgot to mention this in the comment. I think this puts there more logic since home button leads to home, back button elsewhere. Could we check this with Filip?

**Comment by igor-kamil** [comment]
You can write him. By my personal preferance - I would prefer back arrow in section - with the meaning "return".

**Review by igor-kamil — APPROVED** [review]
let's just fix conflicts and release it as it is 🚀 
we can solve "`logo` vs `return-arrow`" dispute later (after testing) 😁

**Pull request merged**